### DISCUSSION TITLE
Ability to configure a list of AWS Regions

### DISCUSSION THREAD

**Discussion by ainapudi** [grafana/grafana#44941] [Data Sources]
Currently, our users only have access to one region. We want the ability to configure a list of AWS regions.

**Comment by sunker** [comment]
Can you elaborate on this? Do you want the list of regions to be configurable on a 1) per cloudwatch data source level, 2) cloudwatch data source instance level or 3) for any AWS plugin?

**Comment by ainapudi** [comment]
The current default works for the customer, we don’t need this anymore.

**Comment by taleena** [comment]
Locking discussion per above comment.
